src/modm/processing/fiber/scheduler.hpp.in

```diff
@@ -100,6 +118,35 @@ class Scheduler
 	}
 };
 
+%% if multicore
+%% for i in range(num_cores)
+template <>
+{{core_storage[i]}} Fiber<{{i}}>* Scheduler<{{i}}>::last = nullptr;
+template <>
+{{core_storage[i]}} Fiber<{{i}}>* Scheduler<{{i}}>::current = nullptr;
```

Comment:
```suggestion
#ifndef MODM_FIBER_STORAGE_{{i}}
#define MODM_FIBER_STORAGE_{{i}} modm_section(".cpu{{id}}_data")
#endif
MODM_FIBER_STORAGE_{{i}} Fiber<{{i}}>* Scheduler<{{i}}>::current = nullptr;
```
Then you can still customize it via compile flags, but I feel it's not important enough for an lbuild option…